accounts/src/main/java/eu/kevin/accounts/bankselection/usecases/GetSupportedBanksUseCase.kt

```diff
@@ -6,14 +6,24 @@ import eu.kevin.accounts.networking.entities.ApiBank
 internal class GetSupportedBanksUseCase(
     private val bankManager: BankManagerInterface
 ) {
-    suspend fun getSupportedBanks(country: String, authState: String, filter: List<String>): List<ApiBank> {
+    suspend fun getSupportedBanks(
+        country: String,
+        authState: String,
+        filter: List<String>,
```

Comment:
@aurimas-zarskis @GytisBarakauskas  maybe it's worth creating a separate entity called `SupportedBanksFilter` which would contain parameters `banks` (which is a list called filter here) and `isAccountLinkingSupported` instead of adding an additional param to this function?

Reply:
Just did that, looks cleaner.